packages/bech32m/tsconfig.json

```diff
@@ -1,9 +1,5 @@
 {
-  "extends": "tsconfig/base.json",
-  "exclude": ["node_modules", "dist"],
-  "compilerOptions": {
-    "outDir": "dist",
-    "noEmit": false,
-    "declarationMap": false
-  }
+  "extends": "tsconfig/tsup.json",
+  "include": ["src"],
+  "exclude": ["node_modules", "dist"]
 }
```

Comment:
tsup builds use a tsup.json base and simply list include/exclude folders (mostly identical)